ui/intermittent-failures/GraphsContainer.jsx

```diff
@@ -97,20 +97,18 @@ GraphsContainer.propTypes = {
     }),
   ),
   graphTwoData: PropTypes.arrayOf(
-    PropTypes.arrayOf(
-      PropTypes.shape({
-        date: PropTypes.shape({ Date: PropTypes.string }),
-        value: PropTypes.number,
-      }),
-    ),
+    PropTypes.shape({
+      date: PropTypes.shape({ Date: PropTypes.string }),
+      value: PropTypes.number,
+    }),
```

Comment:
This should be an array of two objects, right?

Reply:
Yes, an array of two objects. I will use the one in https://github.com/mozilla/treeherder/pull/6308#discussion_r411733305

Reply:
Done